### PR TITLE
Document how to install the Policy Generator on OpenShift GitOps

### PR DESCRIPTION
**Pull request by mprahl**
Relates:
https://github.com/stolostron/backlog/issues/23775

### CODE REVIEW

**Review by dockerymick — APPROVED** [review]
Job well done Matt! Thanks for adding me as a reviewer and making these changes. I made a couple of small updates. When you have an opportunity, will you re-review this PR. I also ran the link check tool and all is well. Approving @mprahl

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -416,6 +417,122 @@ See the link:https://docs.openshift.com/container-platform/4.9/security/complian
...
+    apiGroups:
+      - cluster.open-cluster-management.io
+    resources:
+      - placements
+      - placements/status
+      - placementdecisions
+      - placementdecisions/status
+----
+
+Additionally, create a `ClusterRoleBinding` resource to grant the OpenShift GitOps service account access to the `openshift-gitops-policy-admin` `ClusterRole`. Your `ClusterRoleBinding` might resemble the following resource:
```
Comment (mprahl) [inline governance/policy_generator.adoc]: To be consistent with the term used in the rest of the doc.
```suggestion
Additionally, create a `ClusterRoleBinding` object to grant the OpenShift GitOps service account access to the `openshift-gitops-policy-admin` `ClusterRole`. Your `ClusterRoleBinding` might resemble the following resource:
```

**Comment by mprahl** [comment]
@dockerymick thanks for review and great edits! I just left one minor suggestion. This is good to merge after that suggestion from my point of view.

**Comment by dockerymick** [comment]
Thanks for your suggestion! I agree and have accepted the commit suggestion. Moving forward with the merge

**Pull request merged**